.travis.yml

```diff
@@ -8,7 +8,7 @@ scala:
 cache:
   yarn: true
   directories:
-    - $HOME/.ivy2/cache
+    - $HOME/.cache/
```

Comment:
Why did you remove the `.ivy/cache`? I still have it on my pc - not sure about the travis containers.

Reply:
I removed it because sbt 1.3+ uses coursier which uses `~/.cache/coursier/v1` to store the cached artifacts.

Reply:
Ah good point. thanks! did not think about that!